cmd/local.go

```diff
@@ -56,7 +69,7 @@ func (l LocalModel) Prepare(channels *turnstile.ChannelMap) {
 		//If so are we configured to overwrite?
 		if l.Nonmem.Configuration.Overwrite {
 			log.Debugf("%s Removing directory %s", l.Nonmem.LogIdentifier(), l.Nonmem.OutputDir)
-			err := RemoveContentsAndDir(l.Nonmem.OutputDir)
+			err := fs.RemoveAll(l.Nonmem.OutputDir)
```

Comment:
Moving back to afero as the target for this since this method was created during troubleshooting for the SGE output stream issue.